tap_wrike/schemas/groups.json

```diff
@@ -0,0 +1,43 @@
+{
+    "type": ["object"],
+    "properties": {
```

Comment:
According to [the doc](https://developers.wrike.com/api/v4/groups/) we are missing a couple properties:
- `myTeam`
- `metadata`
They are both optional

For metadata, I think you only get it if you specify the query parameter `fields=metadata` (see for example the `users` endpoint in `streams.py`) which I think we should do just to get every single possible bit of information possible out of this endpoint!

Reply:
How do you know what metadata to sync specifically? Currently, adding the fields "metadata" only adds an empty array.

For the "myTeam", that's basically a row that shows which team the token's creator (me) is on. Currently it is in the gbq, so that actually a transform I'm going to have to apply to remove it

Reply:
It will sync all metadata, no need to specify which to sync. Take a look at this schema for example: https://github.com/potloc/tap-wrike/blob/main/tap_wrike/schemas/folders.json#L72:L85

And yeah, for `myTeam` you are right, it makes sense to drop it :)

Reply:
If you leave it out of the schema it won't be stored in BigQuery, so no need for a transform 🙂